src/types/feature-flag.ts

```diff
@@ -0,0 +1,20 @@
+import type { FeatureState, FlagStatus } from '~/constants/feature-flag'
+
+interface BaseFeatureFlag {
+  status: FlagStatus
+  description?: string
+  data?: object
```

Comment:
`object` is kind of a weird type in TypeScript and generally is safer to avoid it: https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript

This is probably better typed as `unknown` to force users of a flag to have to cast the data to the type they think it should be for the given flag name.

Reply:
Resolved in 90b2a37.